assets/js/admin/admin.js

```diff
@@ -43,31 +43,34 @@ jQuery( document ).ready( function( $ ) {
 			option.innerText = i;
 			$day.append( option );
 		}
-		if ( dayVal < dayCount ) {
```

Comment:
@jhung This was the issue with https://github.com/platform-coop-toolkit/learning-commons-framework/pull/112#discussion_r330608873.